docs/manual/cli/user.en.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "contao:user"
+description: "Create and list Contao users."
+aliases:
+    - /en/cli/user/
+---
+
+
+After the [database](/en/cli/migrate/) has been created, Contao [users](/en/user-management/users/) can be created here. You can get 
+an overview of the commands via:
+
+```bash
+php vendor/bin/contao-console contao:user --help
+```
+
+## contao:user:list
+
+Lists existing Contao users:
+
+| Option | Beschreibung |
+| --- | --- |
+| `--help`   | Options overview |
+| `--admins` | Show administrators only |
+
+```bash
+php vendor/bin/contao-console contao:user:list --help
+```
+
+
+## contao:user:create
+
+Creates a Contao user:
+
+| Option | Beschreibung |
+| --- | --- |
+| `--help`   | Options overview |
+| `--username` | User name |
+| `--name` | Full user name |
+| `--email` | The user e-mail address |
+| `--password` | User password |
+| `--admin` | User is administrator |
+| `--group` | Groups to be assigned to the user |
+| `--change-password` | Prompt users to change their password when they first log in to the backend |
+
+```bash
+php vendor/bin/contao-console contao:user:create --help
+```
+
+
+## contao:user:password
+
+Change of a user password:
+
+```bash
+php vendor/bin/contao-console contao:user:password --help
+```
```

Comment:
```suggestion
Allows modifiying the password for a user

| Option | Description |
| --- | --- |
| `--help`   | Options overview |
| `--username` | The username of the back end user |
| `--password`** | The new password (using this option is not recommended for security reasons) |
| `--p` | The new password (using this option is not recommended for security reasons) |
| `--require-change` | Require the user to change the password on their next login. |
| `--r` | Require the user to change the password on their next login. |

Example usage:
``
php vendor/bin/contao-console contao:user:password  -p mysecurepassword -r k.jones
``
```

Reply:
For security reasons you should not pass the password as an option directly on the CLI, since the password will then be stored in plain text in the hash history.

Reply:
Yup, better leave it undocumented then.
That's why I added the `(using this option is not recommended for security reasons)`

Reply:
The option itself is fine, you should just not use it directly on the CLI.